src/postprocess.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * Postprocess step. Stolen from babel-jest: https://github.com/facebook/jest/blob/master/packages/babel-jest/src/index.js
```

Comment:
'Based on' sounds better than 'Stolen' :)
Also, it might be better to the file at a particular commit. So even if the file changes in the future, we'll have a reference to what this code was based on.